.circleci/tests.unit1.targets

```diff
@@ -0,0 +1,29 @@
+#  Copyright (c) 2021 Hartmut Kaiser
```

Comment:
Would a simple split into algorithms and non-algorithms be enough? This seems very error prone as it requires adding a new module to one of these lists explicitly.

Reply:
No, that still times out. I tried.